test/web-platform-tests/to-run.yaml

```
@@ -845,7 +843,7 @@ DocumentOrShadowRoot-prototype-elementFromPoint.html: [fail, offsetTop not imple
 Element-interface-attachShadow-custom-element.html: [fail, CustomElement.define is not implemented]
 MouseEvent-prototype-offsetX-offsetY.html: [fail, offsetTop not implemented]
 Range-prototype-insertNode.html: [fail, Range is not implemented]
-ShadowRoot-interface.html: [fail, ShadowRoot.innerHTML is still at the stage of draft]
+ShadowRoot-interface.html: [fail, ShadowRoot.styleSheet and ShadowRoot.activeElement are not yet implemented]
```

```suggestion
ShadowRoot-interface.html: [fail, shadowRoot.styleSheet and shadowRoot.activeElement are not yet implemented]
```

(Preexisting problem, but good to fix...)